libs/components/inline-form/src/lib/modules/inline-form/fixtures/inline-form.fixture.ts

```diff
@@ -20,5 +20,7 @@ export class SkyInlineFormFixtureComponent {
 
   public showFormWithNoElements = false;
 
-  public onClose(event: SkyInlineFormCloseArgs): void {}
+  public onClose(event: SkyInlineFormCloseArgs): void {
+    console.log('sky-inline-form onClose event = ', event);
```

Comment:
Lets just `return;` and disable the event warning.

Reply:
done in: [remove console.log. ignore eslint warning](https://github.com/blackbaud/skyux/pull/623/commits/9c5a45c7e29e6a89b3ed96176d3a50bd4bb56496)